tests/utils/index.ts

```diff
@@ -225,6 +225,20 @@ export const getVersionsForTest = (
 export const skipOS = (excludedOS: string[]) => (_version?: string) =>
   excludedOS.length === 0 || excludedOS.includes(process.platform);
 
+/**
+ * Helper callback that skips a test if the CPU arch is included in excludedCPU.
+ * Intended to be passed to `skipTestIf`.
+ */
+export const skipCPU = (excludedCPU: string[]) => (_version?: string) =>
+  excludedCPU.length === 0 || excludedCPU.includes(process.arch);
+
+/**
+ * Helper callback that skips a test if the OS and CPU arch is included in excludedCPU and excludedOS.
+ * Intended to be passed to `skipTestIf`.
+ */
+export const skipCPUOS = (excludedOS: string[], excludedCPU: string[]) => (_version?: string) =>
+  excludedCPU.includes(process.arch) && excludedOS.includes(process.platform);
```

Comment:
I think it would be better if this were pairwise or struct'd somehow. E.g.:

`skipCPUOS([{os: "linux", cpu: "aarch64"}]);`

Not a strong opinion, but I think it would allow us to hit contrasting cases more easily and cleanly